packages/mailer/handlers/resend/src/index.ts

```diff
@@ -0,0 +1,80 @@
+import { Resend } from 'resend'
+import type { Tag } from 'resend/build/src/interfaces'
+
+import type {
+  MailSendOptionsComplete,
+  MailRenderedContent,
+  MailResult,
+} from '@redwoodjs/mailer-core'
+import { AbstractMailHandler } from '@redwoodjs/mailer-core'
+
+export type ResendMailHandlerOptions = {
+  tags?: Tag[]
+}
+
+export class ResendMailHandler extends AbstractMailHandler {
+  private client: Resend
+
+  constructor({ apiToken }: { apiToken: string }) {
+    super()
+    this.client = new Resend(apiToken)
+  }
+
+  async send(
+    content: MailRenderedContent,
+    sendOptions: MailSendOptionsComplete,
+    handlerOptions?: ResendMailHandlerOptions
+  ): Promise<MailResult> {
+    // I was not having success at passing attachment contents as strings directly
+    // to the Resend client, so I'm going to transform them to Buffers if they are
+    // strings.
+    const transformedAttachments = []
```

Comment:
Do you think other mail handlers will have a similar need? If so, break out into a util function? Can do that refactor later perhaps, unless easier to test individual function to convert.

Reply:
Not sure. In theory resend allows the string type but passing them in directly was confusing when it was malformed in my basic tests. 

Happy to either extract it out or even change the interface (not user facing but handler facing) to adapt to what we find more convenient as we add more handlers.

Reply:
Ah, ok. Let's refactor/revisit when implementing other handlers like Postmark, etc etc